Core/Source/MVVM/FilteredModelCollection.swift

```diff
@@ -198,9 +148,11 @@ public final class FilteredModelCollection: ModelCollection, ProxyingCollectionE
                 return
             }
 
```

Comment:
while you're in here, want to add an assert about main thread in this `updateCollection` closure?